src/main/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsView.java

```diff
@@ -0,0 +1,546 @@
+                if ((viewModel.currentAbbreviationProperty().get() != null)
+                        && !viewModel.currentAbbreviationProperty().get().isPseudoAbbreviation()) {
+                    viewModel.deleteAbbreviation();
+                }
+            }
+        });
+        journalTableNameColumn.setCellValueFactory(cellData -> cellData.getValue().nameProperty());
+        journalTableNameColumn.setCellFactory(cell -> new JournalAbbreviationsNameTableEditingCell());
+        journalTableAbbreviationColumn.setCellValueFactory(cellData -> cellData.getValue().abbreviationProperty());
+        journalTableAbbreviationColumn.setCellFactory(cell -> new JournalAbbreviationsAbbreviationTableEditingCell());
+        journalTableEditColumn.setCellValueFactory(cellData -> cellData.getValue().isPseudoAbbreviationProperty());
+        journalTableDeleteColumn.setCellValueFactory(cellData -> cellData.getValue().isPseudoAbbreviationProperty());
+        journalTableEditColumn.setCellFactory(column -> new TableCell<AbbreviationViewModel, Boolean>() {
+
+            @Override
+            protected void updateItem(Boolean isPseudoAbbreviation, boolean isEmpty) {
+                super.updateItem(isPseudoAbbreviation, isEmpty);
+                if (isPseudoAbbreviation != null) {
+                    if (!isEmpty) {
+                        if (isEditableAndRemovable.get()) {
+                            if (!isPseudoAbbreviation) {
+                                Text graphic = new Text(IconTheme.JabRefIcon.EDIT.getCode());
+                                graphic.getStyleClass().add("icon");
+                                setGraphic(graphic);
+                                setOnMouseClicked(evt -> {
+                                    editAbbreviation();
+                                });
+                            } else {
+                                Text graphic = new Text(IconTheme.JabRefIcon.ADD.getCode());
+                                graphic.getStyleClass().add("icon");
+                                setGraphic(graphic);
+                                setOnMouseClicked(evt -> {
+                                    addAbbreviation();
+                                });
+                            }
+                        }
+                    }
+                } else {
+                    setGraphic(null);
+                }
+            }
+        });
+        journalTableDeleteColumn.setCellFactory(column -> new TableCell<AbbreviationViewModel, Boolean>() {
+
+            @Override
+            protected void updateItem(Boolean isPseudoAbbreviation, boolean isEmpty) {
+                super.updateItem(isPseudoAbbreviation, isEmpty);
+                if (isPseudoAbbreviation != null) {
+
+                    if (!isEmpty) {
+                        if (isEditableAndRemovable.get()) {
+                            if (!isPseudoAbbreviation) {
```

Comment:
But I only need to call `setGraphic(null)` on two of the four if statements. However I can join the two if statements with no else block.